app/src/main/java/com/giraffe/cineverseapp/main_activity/MainActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.giraffe.cineverseapp.main_activity
```

Comment:
rename the package to `main`

```suggestion
package com.giraffe.cineverseapp.main
```